### ISSUE TITLE
How to disable head tracking (gyroscope)?

### ISSUE THREAD

**Issue by umbrella-alexey-pak** [googlevr/gvr-ios-sdk#325]
Hi, is there a way to programmatically enable or disable head tracking (gyroscope) in the GVRKit? 
Also in the issue https://github.com/googlevr/gvr-ios-sdk/issues/281 I've read that the GVRKit was open source but the reference in the issue was broken and I didn't found source code of GVRKit in the repository.

**Comment by Performador** [comment]
You should be able to use [gvr->PauseTracking()](https://developers.google.com/vr/reference/ios-ndk/class/gvr/gvr-api#classgvr_1_1_gvr_api_1a99c4d9673162b2f774c570c480ede257).

**Comment by Performador** [comment]
We distribute GVRKit as a compiled framework. You should be able to see the headers in XCode though.

**Comment by umbrella-alexey-pak** [comment]
Could I call this method if I use (GVRSceneRenderer + GVRVideoRenderer) or I have to write my own Renderer with my own gvr_context as shown in TreasureHuntNDK example?

**Comment by umbrella-alexey-pak** [comment]
Could give me some tip? I don't understand how to get a reference or pointer to GvrApi instance.

**Comment by umbrella-alexey-pak** [comment]
No. I'm using (GVRRendererViewController + GVRRendererView + GVRSceneRenderer + GVRVideoRenderer).

**Comment by Performador** [comment]
OK, will add this on GVRRenderer so you can pause tracking.

**Comment by Performador** [comment]
Actually [GVRRenderer pause:NO] should disable tracking.
